src/components/common/NetworkSelector/styles.module.css

```diff
@@ -28,9 +28,9 @@
   pointer-events: none;
 }
 
-.menuItem {
-  padding-top: var(--space-1);
-  padding-bottom: var(--space-1);
+.select :global .MuiMenuItem-root {
+  padding: 0;
+  pointer-events: none;
```

Comment:
Why do we need to set pointer-events: none?

Reply:
when you add this `renderValue` prop to Select it wraps each item in an li and adds a hover highlight